src/Microsoft.DotNet.Build.Tasks/AddDependenciesToProjectJson.cs

```diff
@@ -198,15 +195,34 @@ private void AddPackageItemsToDictionary(ref Dictionary<string, PackageItem> pac
             }
         }
 
-        // A versions file is of the form https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/Latest_Packages.txt
+        private static async System.Threading.Tasks.Task<string> GetFileAsync(string uri)
+        {
+            string contents = string.Empty;
+            using (var stream = (File.Exists(uri)) ? File.OpenRead(uri) : await new HttpClient().GetStreamAsync(uri))
+            using (StreamReader reader = new StreamReader(stream))
+            {
+                contents = await reader.ReadToEndAsync();
+            }
+            return contents;
+        }
+
+        // A versions file is of the form https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/master/Latest_Packages.txt
```

Comment:
"That" kind of Latest is exactly what we need here.  Unless the URI we point to contains the old packages plus whatever is newly published, we will be in a state where there is a gap between LKG and what we published in the interim builds between then and the current servicing release.  How much effort is it to produce that kind of Latest file?  We really need that to make this a workable solution.